app/views/refinery/pages/home.html.erb

```diff
@@ -5,50 +5,46 @@
-        </div>
+      <div class="action-boxes">
+        <a href="/find-out"><span class="wrapper"><%= t('refinery.home.actions.learn') %></span></a>
+        <a href="/stories/new"><span class="wrapper"><%= t('refinery.home.actions.engage') %></span></a>
+        <a href="/events"><span class="wrapper"><%= t('refinery.home.actions.events') %></span></a>
+      </div>
       <% end %>
     </section>
   </div>
-
   <% if @page.content_for(:bottom) %>
-    <div class="container">
-      <section class="content-column">
-        <%= raw @page.content_for(:bottom) %>
-      </section>
-    </div>
+  <div class="container">
+    <section class="content-column">
+      <%= raw @page.content_for(:bottom) %>
+    </section>
+  </div>
   <% end %>
-
   <div class="interruption">
     <div class="container">
       <div class="content-column-row">
-        <div class="content-column-one next-event-floater">
-          <% if @next_event && Flipper.enabled?(:next_event) %>
-            <h2><%= t('refinery.home.interruption.next_event') %></h2>
-            <%= link_to(refinery.events_event_path(@next_event)) do %>
-              <div class="next-event-time hidden-text"><%= @next_event.start.strftime("%B %-d, %Y " + t('refinery.home.interruption.event_at') + " %-l:%M%p") %></div>
-              <div class="next-event-time" aria-hidden="true"><%= @next_event.start.strftime("%b %-d %-l:%M%P") %></div>
-              <div class="next-event-title"><%= @next_event.title %></div>
-            <% end %>
-          <% end %>
+        <div class="content-column-one__image--mc-arrow">
+          <%= image_tag("mc-arrow.png", alt: t('logo_alt')) %>
         </div>
-        <div class="content-column-two">
-          <h2><%= t('refinery.home.email_sign_up.sign_up') %></h2>
-          <a class="button" href="<%= CONSTANT_CONTACT_LANDING_PAGE %>" target="_blank"><%= t('refinery.home.email_sign_up.join') %></a>
+        <div class="content-column-one--regional-plan-text">
+          <h1 class="content-column-two--regional-plan-text content-column-two--regional-plan-title">What is a Regional Plan?</h1>
+          <p>It’s a set of actions that local and state government can take to improve the region over time. Plans
+            like MetroCommon can include thinking about many different aspects of our lives. That’s because policies and laws can affect jobs, transportation, health, and more.</p>
+            <p>MetroCommon will focus on challenges and opportunities that
+            are bigger than any one place. And it will look at how those issues are connected. Topics like traffic
+            congestion, the rising cost of housing, our changing climate, equity of wealth and health, and efficient government – all will be part of the mix.    
+            </p>
+          <a class="content-column-two__button--learn" href="https://youtu.be/bApghSqlbYc" target="_blank">LEARN</a>
```

Comment:
What about making this `Learn More` to conform with the design suggestion:
<img width="342" alt="image" src="https://user-images.githubusercontent.com/565647/54458629-d5135500-473a-11e9-9b3e-ecb29f603bbd.png">

and then use `text-transform: uppercase;` as a style for `content-column-two__button--learn`

Also on my browser font-family is loading as SwiftNeueLTPro, and the template suggests the button should be Calibre Semibold.

Reply:
Also would you consider adding a little padding/margin to the top of the button to separate it from the paragraph?

Reply:
I've added divs in `home.html.erb` and  css classes in `home.scss` to separate the button and its text, allowing the button to have top padding. 

I see what happened now with the "LEARN MORE" button.  The file shows just the word "LEARN" until you expand the button selector handle a little bit to the right, so now I see the full text intended for the button.  I've updated the text and implemented your suggestions.